tests/test_mathml2sympy.py

```diff
@@ -154,6 +179,37 @@ def test_diff_eq(self):
                           '></apply>',
                           [sympy.Eq(sympy.Derivative(V(t), t), -(i_Stim+i_Na+i_K+i_L) / Cm)])
 
+    def test_scientific_notation(self):
+        self.assert_equal('<cn type="e-notation">1.234<sep/>5</cn>', [sympy.Number(1.234e5)])
+
+    def test_xor(self):
+        self.assert_equal('<apply><xor/><ci>a</ci><ci>b</ci></apply>',
+                          [sympy.Xor(sympy.Symbol('a'), sympy.Symbol('b'))])
+
+    def test_not(self):
+        self.assert_equal('<apply><not/><ci>a</ci></apply>',
+                          [sympy.Not(sympy.Symbol('a'))])
+
+    def test_ceiling(self):
+        self.assert_equal('<apply><ceiling/><ci>a</ci></apply>',
+                          [sympy.ceiling(sympy.Symbol('a'))])
+
+    def test_min(self):
+        self.assert_equal('<apply><min/><ci>a</ci><ci>b</ci></apply>',
+                          [sympy.Min(sympy.Symbol('a'), sympy.Symbol('b'))])
+
+    def test_max(self):
+        self.assert_equal('<apply><max/><ci>a</ci><ci>b</ci></apply>',
+                          [sympy.Max(sympy.Symbol('a'), sympy.Symbol('b'))])
+
+    def test_trig(self):
```

Comment:
Are we not doing the full trig set cellml supports? So (arccos hyperbolicus etc. :p)

Reply:
They're there but only tested a few.